carma-messenger-core/j2735_convertor/src/control_message_convertor.cpp

```diff
@@ -279,8 +279,10 @@ void convert(const j2735_v2x_msgs::msg::TrafficControlSchedule& in_msg, carma_v2
 
 void convert(const carma_v2x_msgs::msg::DailySchedule& in_msg, j2735_v2x_msgs::msg::DailySchedule& out_msg)
 {
-  out_msg.begin = in_msg.begin.sec / units::SEC_PER_MIN;
-  out_msg.duration = in_msg.duration.sec / units::SEC_PER_MIN;
+  rclcpp::Time begin((in_msg.begin.sec + in_msg.begin.nanosec), 0.0);
```

Comment:
This looks wrong. You are adding primitives here. I believe it should be
```
rclcpp::Time begin(in_msg.begin.sec,  in_msg.begin.nanosec);
```
Double check the API docs in case its the other way around.